app/build.gradle

```diff
@@ -48,9 +50,25 @@ dependencies {
     annotationProcessor libraries.immutablesValue // <-- for annotation processor
     compileOnly libraries.immutablesValue // <-- for annotation API
     compileOnly libraries.immutablesGson // for annotations
-    implementation 'com.nytimes.android:store3:3.0.1'
-    implementation 'com.nytimes.android:cache3:3.0.1'
-    implementation 'com.nytimes.android:middleware3:3.0.1'
-    implementation 'com.nytimes.android:filesystem3:3.0.1'
+    implementation project(':store')
+    implementation project(':cache')
+    implementation project(':middleware')
+    implementation project(':filesystem')
     implementation libraries.rxAndroid2
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+
+    implementation "android.arch.persistence.room:runtime:$room_version"
+    annotationProcessor "android.arch.persistence.room:compiler:$room_version"
+    kapt "android.arch.persistence.room:compiler:$room_version"
+
+//    androidTestImplementation "android.arch.persistence.room:testing:$room_version"
+    // optional - RxJava support for Room
+    implementation "android.arch.persistence.room:rxjava2:$room_version"
 }
+repositories {
+    mavenCentral()
+}
+
+configurations.all {
+    resolutionStrategy.force 'com.android.support:support-v4:26.1.0'
```

Comment:
It seems we're still using support lib v26.1.0 while there is latest one v27.1.1. Any plan to upgrade?